mm2src/common/common.rs

```diff
@@ -1014,3 +1015,8 @@ pub fn sha256_digest(path: &PathBuf) -> Result<String, std::io::Error> {
     };
     Ok(digest)
 }
+
+pub fn parse_rfc3339_to_timestamp(date_str: &str) -> Result<u64, ParseError> {
+    let date: DateTime<Utc> = date_str.parse()?;
+    Ok(date.timestamp() as u64)
```

Comment:
i64 to u64 might cause integer overflow, consider using `try_into` or one of the `number_type_casting` macros

Reply:
added try_into, thanks!